packages/webpack-plugin/src/mini-css-support.ts

```diff
@@ -1,23 +1,29 @@
 import type { Compilation, Compiler, NormalModule } from 'webpack';
 import { replaceMappedCSSAssetPlaceholders, getStylableBuildData } from './plugin-utils';
 
-const memoize = require('webpack/lib/util/memoize');
-
 import { StylableWebpackPlugin } from './plugin';
 import { BuildData } from './types';
 
-const getMiniCssExtractPlugin = memoize(() => {
-    return require('mini-css-extract-plugin');
-});
 export function injectCssModules(
     webpack: Compiler['webpack'],
     compilation: Compilation,
     staticPublicPath: string,
     stylableModules: Map<NormalModule, BuildData | null>,
     assetsModules: Map<string, NormalModule>
 ) {
-    const MiniCssExtractPlugin = getMiniCssExtractPlugin();
-    const CssModule = MiniCssExtractPlugin.getCssModule(webpack);
+    const MiniCssExtractPlugin = compilation.options.plugins.find(
+        (plugin) => plugin.constructor.name === 'MiniCssExtractPlugin'
```

Comment:
this assumes the plugins are all classes, but they might be objects with apply I think. not sure though. might wanna protect against constructor not being available.